src/sdk/namespace_s3.js

```diff
@@ -90,14 +90,31 @@ class NamespaceS3 {
         dbg.log0('NamespaceS3.list_objects:', this.bucket, inspect(params),
             'list', inspect(res));
 
+        let bucket_info;
+        try {
+            bucket_info = await object_sdk.read_bucket_sdk_config_info(params.bucket);
+        } catch (err) {
+            dbg.warn('list_buckets: read_bucket_sdk_config_info of bucket', params.bucket, 'got an error', err);
+            // in case the config file was deleted during the bucket list - we will continue
+            if (err.rpc_code !== 'NO_SUCH_BUCKET') throw err;
+        }
+
         return {
             objects: _.map(res.Contents, obj => this._get_s3_object_info(obj, params.bucket)),
             common_prefixes: _.map(res.CommonPrefixes, 'Prefix'),
             is_truncated: res.IsTruncated,
             next_marker: res.NextMarker,
+            object_owner: this.get_object_owner(bucket_info),
         };
     }
 
+    get_object_owner(info) {
+        return {
+              name: info.bucket_owner.unwrap(),
+              id: info._id,
```

Comment:
This is the id of the bucket? we need to have the id of the bucket_owner account

Reply:
Both ids are coming same. Let me assing owner id only

Reply:
owner account id and bucket id is the same?

Reply:
Yes.

`"_id":"668d0822dafa670026921b64","name":"tito-1f04b041-673f-4883-a83c-056bcc6a99af","active_triggers":[],"system_owner":"admin@noobaa.io","bucket_owner":"operator@noobaa.io","bucket_info":{"name":"tito-1f04b041-673f-4883-a83c-056bcc6a99af","owner_account":{"email":"operator@noobaa.io","id":"668d0693dafa670026921b54"}`

Reply:
bucket id - `668d0822dafa670026921b64`
owner_account._id - `668d0693dafa670026921b54`
it's not the same